services/core-api/app/api/projects/project_summary/resources/project_summary_list.py

```diff
@@ -1,23 +1,53 @@
+from flask import request
 from flask_restplus import Resource, inputs
-from werkzeug.exceptions import NotFound
+from werkzeug.exceptions import NotFound, BadRequest
 from datetime import datetime, timezone
-from decimal import Decimal
 
 from app.extensions import api
 from app.api.utils.resources_mixins import UserMixin
 from werkzeug.exceptions import InternalServerError
 from app.api.utils.custom_reqparser import CustomReqparser
 from app.api.utils.access_decorators import MINE_ADMIN, requires_any_of, VIEW_ALL, MINESPACE_PROPONENT, MINE_EDIT, is_minespace_user
 from app.api.mines.mine.models.mine import Mine
-from app.api.mines.project_summary.response_models import PROJECT_SUMMARY_MODEL
-from app.api.mines.project_summary.models.project_summary import ProjectSummary
-from app.api.mines.project_summary.models.project_summary_contact import ProjectSummaryContact
-from app.api.mines.project_summary.models.project_summary_authorization import ProjectSummaryAuthorization
+from app.api.projects.project_summary.response_models import PROJECT_SUMMARY_MODEL
+from app.api.projects.project.models.project import Project
+from app.api.projects.project_summary.models.project_summary import ProjectSummary
 
 
-class ProjectSummaryListResource(Resource, UserMixin):
+class ProjectSummaryListGetResource(Resource, UserMixin):
```

Comment:
I would also like to chat on Tuesday around these resources and why there is a distinct difference between the get and post resource.

Reply:
Much like most of your other comments, the reason some of this looks odd is because Minespace doesn't have  a Project space yet in the UI meaning we don't have access to the project_guid in many circumstances, these are all temporary measures and will be addressed when the work on https://bcmines.atlassian.net/browse/MDS-4261 begins. Anywho I can walk you through that tomorrow.